programs/cli_wallet/main.cpp

```diff
@@ -72,6 +72,9 @@ int main( int argc, char** argv )
 {
    try {
 
+      const char* regex_secret_name = "regex-secret";
+      const string regex_secret = "^\\s*(unlock|set_password)\\s*";
```

Comment:
Why not `^\\s*(unlock|set_password)\\s*$`?

IMHO good UX is:
* if the user typed `unlock<ENTER>`, we prompt for a password;
* if the user typed `unlock SOMETHING<ENTER>`, we use `SOMETHING` as password.

Thoughts?

Reply:
It works now right this way if I remember it correctly ! You should check it to see ...

Reply:
Yes, it's current behavior. Thanks.

I guess the `^` is not needed. Not a big deal though.

Reply:
done